src/sql/src/plan/error.rs

```diff
@@ -451,6 +459,12 @@ impl fmt::Display for PlanError {
             Self::PostgresDatabaseMissingFilteredSchemas { schemas} => {
                 write!(f, "FOR SCHEMAS (..) included {}, but PostgreSQL database has no schema with that name", itertools::join(schemas.iter(), ", "))
             }
+            Self::InvalidKeysInSubscribeEnvelopeUpsert => {
```

Comment:
There does not seem to be a test for either of those two error conditions.

Reply:
Added a test for the invalid within timestamp order by. Reading through the code i don't think it's possible to hit this "invalid keys in envelope upsert" error: we treat the keys we parse as identifiers and identifiers can never generate extra HIR in `map_exprs`. I'm leaning towards keeping this error: I need to handle a non empty `map_exprs` and an error seems nicer than a panic if an error path can happen